src/maestral/utils/appdirs.py

```diff
@@ -9,6 +9,8 @@
 from os import path as osp
 from typing import Optional
 
+from maestral.constants import IS_BSD
```

Comment:
I don't quite remember why I did not import from `maestral.constants` in the first place here, maybe there was a circular dependency somewhere before this module moved.

Could you adapt the other checks accordingly with IS_LINUX and IS_MACOS? Also, please change to a relative import `from ..constants import ...` for consistency and to make the life of static type checkers easier.

Reply:
Hmm... well the reason seems to be that the `test_*_dirs` tests seem to break if you do that!

I shall put it all back to `platform.system()` as before.